### PR TITLE
[Sprint maintain] information user logic #1

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/140246
- information user logic
- Effect : None

### CODE REVIEW

File: app/src/main/res/drawable/ic_arrow_up.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <path
+        android:fillColor="@color/colorPrimary"
+        android:pathData="M434.252,208.708L248.387,22.843c-7.042-7.043-15.693-10.564-25.977-10.564c-10.467,0-19.036,3.521-25.697,10.564
+L10.848,208.708C3.615,215.94,0,224.604,0,234.692c0,9.897,3.619,18.459,10.848,25.693l21.411,21.409
+c6.854,7.231,15.42,10.855,25.697,10.855c10.278,0,18.842-3.624,25.697-10.855l83.939-83.651v200.998
+c0,9.89,3.567,17.936,10.706,24.126c7.139,6.184,15.752,9.273,25.837,9.273h36.545c10.089,0,18.698-3.09,25.837-9.273
+c7.139-6.188,10.712-14.236,10.712-24.126V198.144l83.938,83.651c6.848,7.231,15.413,10.855,25.7,10.855
+c10.082,0,18.747-3.624,25.975-10.855l21.409-21.409c7.043-7.426,10.567-15.988,10.567-25.693
+C444.819,224.795,441.295,216.134,434.252,208.708z"/>
+</vector>
```
Comment (HyperionsThs) [inline app/src/main/res/drawable/ic_arrow_up.xml]: enter end line

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -386,6 +386,7 @@
     <!-- Information Profile -->
     <string name="view_more">View More</string>
     <string name="empty">Empty !</string>
+    <string name="km">km(s)</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`km`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/layout/layout_working_other.xml [hunk 3]
```diff
@@ -180,6 +187,7 @@
                 >
                 <EditText
                     android:hint="@string/bank_info"
+                    bind:setInformationUser="@{viewModel.user.bankInfo.bankName}"
```
Comment (daolq3012) [inline app/src/main/res/layout/layout_working_other.xml]: pls add condition if Object = null new Object inside modle ``User`` ``BankInfo``

Comment (daolq3012) [reply app/src/main/res/layout/layout_working_other.xml]: do the same with the other line!

**Pull request merged**